Snippets/ASQ/ASQ_7/Configuration/MessageWrapperSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ASQ_7.Configuration
+{
+    using NServiceBus;
+
+    public class MessageWrapperSerializer
+    {
+        #region SerializerAndMessageWrapperSerializer
+        public void ConfigureEndpointA()
+        {
+            var endpointA = new EndpointConfiguration("Endpoint-A");
+            // serialize the messages using the XML serializer:
+            endpointA.UseSerialization<XmlSerializer>();
```

Comment:
I can't imagine using another serializer for the message body for any other reason than the payload size, so maybe `Xml` isn't the best option here.